doc/release-notes/8983-more-than-50000-entries-in-sitemap.md

```diff
@@ -0,0 +1,3 @@
+The sitemap file generation can handle more than 50,000 entries if needed with the [sitemapgen4j](https://github.com/gdcc/sitemapgen4j) library, maintained by the Global Dataverse Community Consortium.
+
+In this case, the Dataverse Admin API `api/admin/sitemap` create a sitemap index file, called `sitemap_index.xml`, in place of the `sitemap.xml` file. See the config section of the Installation Guide for details.
```

Comment:
Could you please add just a couple of words here, along the lines of "... in place of the single `sitemap.xml`, which will be split into individual files `sitemap_1.xml`, `sitemap_2.xml` etc. as needed; these files will be listed in the index files and placed in the same directory". I obviously encourage you to try and explain this better and in a more clear way (hence "along the lines of ..."

Reply:
Hi @landreev.

Thanks you for the review. A update of release notes snippet and configuration page have been done, hoping to have taken your feedback.